### PR TITLE
Add new name to alternative file

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1342
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/117
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/156

### CODE REVIEW

File: Media/DisplayMedia/Strategies/ImageStrategy.php [hunk 1]
```diff
@@ -65,7 +65,13 @@ public function displayMediaForWysiwyg(MediaInterface $media, $format = '')
      */
     public function getMediaFormatUrl(MediaInterface $media, $format)
     {
-        return $this->getFileUrl($media->getFilesystemName(), $format);
+        $key = $media->getFilesystemName();
```
Comment (SOLenG) [inline Media/DisplayMedia/Strategies/ImageStrategy.php]: I I would have rather seen in an else block

Comment (itkg-ngilain) [inline Media/DisplayMedia/Strategies/ImageStrategy.php]: @SOLenG : in this project we avoid as possible the 'else' statments. You can ask @nicolasThal for info about that.

**Comment by SOLenG** [comment]
:+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**